### DISCUSSION TITLE
qTranslate XT does not translate Germanized WooCommerce custom checkout fields

### DISCUSSION THREAD

**Discussion by HSFLAG** [qtranslate/qtranslate-xt#1296] [Q&A]
Good day

We have been using Vendidero Germanized Plugin and qtranslate-XT for years with two Custom Legal checkboxes for the WooCommerce Checkout (respectively Terms & Conditions and Consent for Media Partners). 

However, since the last Germanized & Pro update, the checkboxes are no longer translated correctly after the WooCommerce Checkout page is being reconstructed by WooCommerce.  The Germanized team mentioned that they reload all content via AJAX when parts of the Checkout page are refreshed by WooCommerce but it seems the qTranslateXT translation does not occur anymore for those fields only when refreshing that part of the page. For other checkout fields created with other plugins, the translation works. 

What happens:

At moment 1) When the "Checkout" web page is loaded, the Custom Legal Checkboxes are displayed correctly for a very short moment.... (see screenshot with cell phone)
https://hsflch-my.sharepoint.com/:i:/g/personal/cristian_duda_hsfl_ch/ERitU6XxljlKvoYYM6jQ7BYBejZ0FyIG3k5qKWrCCqufXA?e=ZGcV3g
and then...

At moment 2) a refresh occurs and suddenly both EN + DE texts are shown, in the final version of the website. 
https://hsflch-my.sharepoint.com/:i:/g/personal/cristian_duda_hsfl_ch/Ebu0F0yNHKtKoxdI98gu82cBIKBodLT9mLhx5diP7iQVGw?e=9gHYAK

Until now, the text was always displayed correctly and translated. What can be done to fix this translation? 

Thanks in advance and kind regards,
Cristian

**Comment by mwie1409** [comment]
Hi Christian,

have you solved the issue in the meantime?
I encountered the same problem and here is my solution:

Within your themes folder under /wp-content/<theme> create these folders:
/wp-content/themes/<theme>**/woocommerce-germanized/checkout**

Create a file called terms.php.

Add the attached code into terms.php.
[terms.txt](https://github.com/qtranslate/qtranslate-xt/files/14296801/terms.txt)

This is a copy of the orginal terms.php from Germanized but with the correct code to enable translation of the checkbox text:
`<?php _e( $checkbox->get_label(), 'woocommerce-germanized' ); ?>`

Best,
Michael
